### PR TITLE
ZK-3924: stuck notification message

### PR DESCRIPTION
**Pull request by klyvechen**

### CODE REVIEW

File: zul/src/archive/web/js/zul/wgt/Notification.js [hunk 1]
```diff
@@ -161,7 +163,7 @@ zul.wgt.Notification = zk.$extends(zul.wgt.Popup, {
 
 		var pzcls = this.$s('pointer'),
 			n = this.$n(),
-			refn = ref.$n(),
+			refn = ref.$n();
 			dir = this._dir,
```
Comment (DevChu) [inline zul/src/archive/web/js/zul/wgt/Notification.js]: global?

Comment (klyvechen) [reply zul/src/archive/web/js/zul/wgt/Notification.js]: sorry, I test the code before and try to change back, but not change it completely.

**Comment by DevChu** [comment]
@zkadmin rebase

**Comment by zkadmin** [comment]
I just pushed up the changes, enjoy!

**Pull request merged**